src/studio/src/designer/frontend/shared/features/dataModelling/DataModelling.tsx

```diff
@@ -151,9 +163,43 @@ function DataModelling({
   }
 
   const shouldDisplayLandingPage = landingDialogState === LandingDialogState.DialogIsVisible;
+  const [hideIntroPage, setHideIntroPage] =React.useState(() => {
+    const datamodelLocalStorage = (JSON.parse(localStorage.getItem('datamodelLocalStorage')) as any);
+    if (datamodelLocalStorage) return datamodelLocalStorage.hideIntroPage;
+    return false;
```

Comment:
Jeg ville nok hatt noe kode rundt localstorage som hadde tatt seg av serialisering og deserialisering. Er langt enklere å mocke ut i test også, samt teste individuelt.

Ville også unngått å hatt `datamodelLocalStorage` som en slik magisk konstant, men heller lagt den til i en enum eller noe annet som gjør at vi kan refaktorere denne.

Reply:
Enig! Opprettet issue https://github.com/Altinn/altinn-studio/issues/8989 for å følge opp